subsys/net/lib/lwm2m/Kconfig

```diff
@@ -264,6 +264,32 @@ config LWM2M_FIRMWARE_UPDATE_PULL_SUPPORT
 	  block transfer and "FIRMWARE PACKAGE URI" resource.  This option
 	  adds another UDP context and packet handling.
 
+config LWM2M_SWMGMT_OBJ_SUPPORT
+	bool "Software management object support"
+	default n
```

Comment:
nit: We typically avoid `default n` since it's the implicit value.

Reply:
I'm a bit confused here. This change is in response to this comment https://github.com/zephyrproject-rtos/zephyr/pull/40552#discussion_r768668057
Is there a different way to turn this off without `default n`?

Reply:
Removing that `default n` line will result as same behaviour.

Reply:
Just delete the `default n` from the option, `default n` is the same as if there's no default at all, so since it's redundant we avoid it (tree-wise, it's not an lwm2m thing).

Reply:
ok, fixed